Sandbox/Sandbox/LoginWithPasswordController.swift

```diff
@@ -20,10 +23,75 @@ class LoginWithPasswordController: UIViewController {
             .loginWithPassword(email: email, phoneNumber: phoneNumber, customIdentifier: customIdentifier, password: password, origin: "LoginWithPasswordController.loginWithPassword")
             .onSuccess { token in
                 self.error.text = nil
-                self.goToProfile(token)
+                if(token.token == nil) {
+                    self.goToProfile(token)
+                } else {
+                    let selectMfaAuthTypeAlert = UIAlertController(title: "Select MFA", message: "Select MFA auth type", preferredStyle: UIAlertController.Style.alert)
+                    guard let stepUpToken = token.token else {
```

Comment:
We're already in the `else` branch of the condition line 26, it's kind of sad this is necessary.